tests/test_inventory/group_vars/all.yml

```diff
@@ -0,0 +1,79 @@
+# Directory where the binaries will be installed
```

Comment:
I think you don't an extra group_vars if you put the inventory in the main inventory